ext/openssl/tests/bug79589.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Bug #65538: TLS unexpected EOF failure
+--EXTENSIONS--
+openssl
+--SKIPIF--
+<?php
+if (getenv("SKIP_ONLINE_TESTS")) die("skip online test");
+?>
+--FILE--
+<?php
+
+$release = file_get_contents(
+    'https://chromedriver.storage.googleapis.com/LATEST_RELEASE',
```

Comment:
Can we verify that this is actually missing an EOF? When Google fixes this the test will become useless.

Reply:
How would you verify it? Note it's an online test so those are in general quite useless as they don't run in pipeline (at least they shouldn't) but better than nothing.

Anyway it's more temporary thing I have got a note to try to recreate it using a server proxy test and send FIN but it's a bit longer one to do which I would like to do with optional disabling.

Reply:
We have other tests that depend on external sources not changing, so this is likely fine, it was just a thought I had. And also, a regression here is relatively unlikely.